kernel/safe_typing.ml

```diff
@@ -222,13 +222,6 @@ let inline_private_constants_in_constr = Term_typing.inline_side_effects
 let inline_private_constants_in_definition_entry = Term_typing.inline_entry_side_effects
 let side_effects_of_private_constants x = Term_typing.uniq_seff (List.rev x)
 
-let constant_entry_of_private_constant = function
```

Comment:
Am I correct that this is unrelated dead code? In general, it's better to use a different PR for these, to facilitate review. To help with that, let me introduce [git worktree](https://git-scm.com/docs/git-worktree).

When a different PR is not practical, using a different commit is still desirable as it would facilitate potential reverts.

Just saying. It shouldn't block this PR.